pkg/storage/blockdev.go

```diff
@@ -190,3 +198,27 @@ func FilterEFISystemPartitions(devices []BlockDev) ([]BlockDev, error) {
 	}
 	return esps, nil
 }
+
+// GetMountpointByDevice gets the mountpoint by given
+// device name. Returns on first match
+func GetMountpointByDevice(devicePath string) (string, error) {
+	mountsPath, err := os.Open(LinuxMountsPath)
+	if err != nil {
+		return "", err
+	}
+
+	defer mountsPath.Close()
+	scanner := bufio.NewScanner(mountsPath)
+	scanner.Split(bufio.ScanLines)
+
+	var mountpoint string
+	for scanner.Scan() {
+		deviceInfo := strings.Split(scanner.Text(), LinuxMountsPathDelimiter)
+		if deviceInfo[0] == devicePath {
+			mountpoint = deviceInfo[1]
```

Comment:
just `return deviceInfo[1], nil` at this point (or `return &deviceInfo[1], nil` if we go for a string pointer)

Reply:
Done